tartiflette/schema.py

```diff
@@ -104,8 +104,18 @@ def subscription_type(self) -> Optional[str]:
     def subscription_type(self, value: str) -> None:
         self._subscription_type = value
 
+    @property
+    def types(self):
+        return self._gql_types
+
     def add_definition(self, value: GraphQLType) -> None:
         # TODO: Check stuff, call update_schema after each new definition ?
+        if not isinstance(value, GraphQLType):
+            raise TartifletteSchemaValidationError(
+                'new GraphQLSchema type definition `{}` '
+                'is not a GraphQLType.'.format(
+                    value.__class__.__name__)
+            )
```

Comment:
maybe hanlde the method in a try-except and raise the TfftValidError if AttributeError is raised ?

Reply:
Removed this safety. If someone messes with the schema, it's his responsibility.